triangle.js

```diff
@@ -0,0 +1,37 @@
+function isTriangle (side1, side2, side3) {
+  if (checkZero(side1, side2, side3) === false || checkNegative(side1, side2, side3) === false || checkSides(side1, side2, side3) === false) {
```

Comment:
Could have done `if (!checkZero(..) && !checkNegative(..) && !checkSides(..))` no need to compare to true and false in if statements